pkg/internal/checkup/executor/executor.go

```diff
@@ -61,5 +62,15 @@ func (e Executor) Execute(ctx context.Context, vmiUnderTestName string) (status.
 		return status.Results{}, fmt.Errorf("failed to login to VMI \"%s/%s\": %w", e.namespace, vmiUnderTestName, err)
 	}
 
-	return status.Results{}, nil
+	oslatTest := oslat.NewTest(vmiUnderTestConsoleExpecter, e.OslatDuration)
+	log.Printf("Running Oslat test on VMI under test for %s...", e.OslatDuration.String())
+	maxLatency, err := oslatTest.Run(ctx)
+	if err != nil {
+		return status.Results{}, fmt.Errorf("failed to run Oslat on VMI \"%s/%s\": %w", e.namespace, vmiUnderTestName, err)
+	}
+	log.Printf("Max Oslat Latency measured: %s", maxLatency.String())
+
+	return status.Results{
+		OslatMaxLatency: maxLatency,
```

Comment:
Please an assertion to the E2E test to check that this field is not empty.

Reply:
DONE